nekton/utils/bin.py

```diff
@@ -18,21 +19,21 @@ def make_exec_bin():
     process.communicate()
 
 
-def run_bin(path: str, outpath:str = None):
+def run_bin(path: str, outpath:str = None,ignore_flag:str = None,merge_flag:str = None):
```

Comment:
Please set the default string values here instead of None

Reply:
done